Io test

adampep opened the pull request:
Short io test. For now only p_n-p_n-2, but I don't think there is any reason to run p_-n-p_n

adampep committed:
Added global norm function for pressure mesh

adampep committed:
Removing conflict between pressure and passive scalar naming

adampep committed:
Correcting byte swap for double precission files

adampep committed:
Enabling .fld ascii for writing

adampep committed:
Ensure proper I/O setup with differen runtime parameters

Necessary chages:
- proper file numbering done on all processes
- proper reader node marking

adampep committed:
Added short I/O test

adampep committed:
adjusting test for sierial run and correcting typo

adampep committed:
adding test to unittest

adampep committed:
adding test to travis

stgeke commented:
Thanks Adam!
Instead of checking the code myself ... can you please tell me what combinations are tested.

adampep committed:
Correct conditional for multiple .f files

stgeke commented:
@adampep Can you please resolve the conflicts. Thanks!

stgeke commented:
* All io-tests should produce a test answer (PASSED/FAILED) printed to stdout
* Can you please include a test for gfldr()

adampep committed:
Updated test

1) added PASS/FAIL flag to all tests
2) added gfldr test, however it is commneted for now, as it does not
pass due to large l2 norms

adampep committed:
Merge branch 'master' into io_test
